crates/polars-core/src/frame/groupby/aggregations/mod.rs

```diff
@@ -111,19 +111,15 @@ where
         })
         .collect_trusted::<Vec<_>>();
 
-    Box::new(PrimitiveArray::new(
-        T::PRIMITIVE.into(),
-        out.into(),
-        Some(validity.into()),
-    ))
+    PrimitiveArray::new(T::PRIMITIVE.into(), out.into(), Some(validity.into()))
```

Comment:
Is this going to prevent rolling operations from returning multiple values per window in the future?

Reply:
None of this is preventing anything, I'm just (blindly) rewriting what we have using the safe API. Parts can always be changed to the unsafe API again if we need to.